packages/components/chip/src/Chip.doc.mdx

```diff
@@ -11,14 +12,23 @@ import * as stories from './Chip.stories'
 
 Help people enter information, make selections, filter content, or trigger actions.
 
-Use chips to show options for a specific context.
+### Chips aren’t buttons
 
-Four types of usages:
+<Callout kind="warning">
+  Chips and buttons are similar in that they both provide visual cues to prompt users to take
+  actions and make selections. However, there are important distinctions to be aware of.
+</Callout>
 
-- [Assist](#assist)
-- [Filter](#filter)
-- [Input](#input)
-- [Suggestion](#suggestion)
+<Callout kind="warning">
+  Multiple chips should appear together in a group, whereas there should be no more than 3 buttons
+  in a single arrangement.
+</Callout>
+
+Chips are reactive and contextual, whereas buttons are static and predetermined.
+A chip should offer a different action depending on the nature of the content it supports,
+whereas a button should be a persistent fixture of a layout.
+From a user-flow perspective,
+chips represent forking paths in an experience while a button represents a linear step along a flow.
```

Comment:
This block of text is too complicated imho. Terms like "forking paths" or "persistent fixture", etc are confusing. We should either remove this from the documentation (guidelines are in ZH) or keep it short and simple.